requirements.txt

```diff
@@ -15,6 +15,7 @@ ipynb-py-convert
 ipytree
 matplotlib
 mss
+geopandas
```

Comment:
Please undo this last commit. Geopandas should NOT be included in the requirements.txt. It is challenging to install geopandas using pip. It is an optional dependency. Users can choose to install geopandas by themselves.